virtio-devices/src/balloon.rs

```diff
@@ -233,7 +317,7 @@ impl BalloonEpollHandler {
             let mut descs_len = 0;
             while let Some(desc) = desc_chain.next() {
                 descs_len += desc.len();
-                Self::release_memory_range(desc_chain.memory(), desc.addr(), desc.len() as usize)?;
+                self.release_memory_range(desc_chain.memory(), desc.addr(), desc.len() as usize)?;
```

Comment:
This is not about balloon inflation, its behavior should not be changed.

Reply:
Yeah. I tried but failed. because we should pass pbp to it, but I can't move pbp which is an Option and without Copy trait. Taking release_memory_range as a method can avoid this issue.